File: ruby/pigatize.rb
def pigatize(word)

  vowels = "aeiou"
  word_down = word.downcase
  word_slice = word.slice(1..-1)

  if vowels.include?(word_down[0])
    puts "#{word}" + "ay"
  else
    puts "#{word_slice}#{word[0]}ay"
  end
end

pigatize("apple")
pigatize("banana")

File: javascript/pigatize.js
function pigatize(word) {
  var first_letter = word[0]
  var word_without_first = word.slice(1)
  var vowels = /[aeiou]/

  if (vowels.test(first_letter)) {
    console.log(word + "ay")
  } else {
    console.log( word_without_first + first_letter + "ay")
  };
}

pigatize("apple")
pigatize("banana")
